go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/go-yaml/yaml v2.1.0+incompatible
 	github.com/gorilla/websocket v1.4.2
 	github.com/mr-tron/base58 v1.1.2
-	github.com/nspcc-dev/dbft v0.0.0-20200531081613-7a39e7b757ac
+	github.com/nspcc-dev/dbft v0.0.0-20200619044932-18e0bf5aaa0c
```

Comment:
It's not `go.mod: tidy`, it's changing the version of dbft.

Reply:
And it should probably be done at the same time with `consensus` changes, the library API has changed.